### PR TITLE
Final PR

### PR DESCRIPTION
**Pull request by SimbaWANGU**
Description This a mobile web application to check a list of metrics (numeric values) using React and Redux. An API that provides numeric data about a topic was selected and a web app was built around it. The web app has these pages:

- a home page with a list of items that could be filtered by some parameters.
- another page for the item details

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,65 @@
...
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- [Microverse](https://www.microverse.org/)
+
```
Comment (ShadyShawkat) [inline README.md]: - [x] Kindly give appropriate credit to the author of the template in your README, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=The%20Creative%20Commons%20license%20of%20the%20design%20requires%20that%20you%20give%20appropriate%20credit%20to%20the%20author.%20Therefore%2C%20you%20must%20do%20it%20in%20the%20README%20of%20your%20project.) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @SimbaWANGU, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All tests are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Your readme looks professional ✔️ 
- Good overall design ✔️ 

Image

## Required Changes ♻️
- [x] Please make sure that your showcase video includes the following:
      - a description of the project.
      - a demo of the project features.
      - you should also highlight some interesting piece of code or something you built that you are very proud of.
- Please make sure to talk through the project while showcasing it and also, highlight a piece of code you're proud of and talk about it briefly, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Once%20you%20have,very%20proud%20of.) requirement.

- [x] _Check the comments under the review for more changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SimbaWANGU** [comment]
Thank you for the review @ShadyShawkat 
I will work on these updates 👍

**Review by Gambit142 — APPROVED** [review]
Hi @SimbaWANGU 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

#### Great work implementing the changes requested by another code reviewer. 👍 

ApprovedAgreeGIF


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SimbaWANGU** [comment]
Thank you for the approval @Gambit142

**Pull request merged**